main.py

```diff
@@ -93,23 +99,25 @@ def create_transaction(self):
         #     self.stop()
         #     return
 
-    def check_transactions(self):
-        print(f'[Node {self.get_peer_id(self.my_peer)}] Checking transactions')
-        logging.info(f'[Node {self.get_peer_id(self.my_peer)}] Checking transactions')
 
+    def block_creation(self):
+        # WIP: get every known peer, not only the ones im connected to
+        peers = self.get_peers()
+        peers.append(self.my_peer)
+        selectedPeer = random.choice(peers)
```

Comment:
not sure about this part, can we discuss a code with Bulat tomorrow?

Reply:
yeah, sure